tests/config_id/enc/enc_kss.c

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/report.h>
+#include "../common.h"
+#include "config_id_t.h"
+
+oe_result_t enclave_test_config_id()
+{
+    OE_TRACE_INFO("enclave_config_id_test_kss_properties invoked\n");
+    oe_result_t result = OE_UNEXPECTED;
+    size_t report_size = OE_MAX_REPORT_SIZE;
+    uint8_t* remote_report = NULL;
+    oe_report_header_t* header = NULL;
+    sgx_quote_t* quote = NULL;
+
+    result = oe_get_report(
```

Comment:
Please call the 'evidence' set of the API. I.e.

1. oe_get_evidence()
2. oe_verify_evidence()
3. _find_claim(OE_CLAIM_SGX_CONFIG_ID)
4. _find_claim(OE_CLAIM_SGX_CONFIG_SVN)
5. memcmp(configid)
6. memcmp(configsvn)

Similar check in openenclave\tests\tools\oesign\test-enclave\enclave\enc.c between line 180 and 240.